lessons/1.mdx

```diff
@@ -74,7 +74,7 @@ There are a lot more features in Remix, but those are the ones we’ll be workin
 
 To get started we will use the Remix IDE to write, compile, test and debug code quickly and easily. Head over to [https://remix.ethereum.org/](https://remix.ethereum.org/) to see what it’s all about and get started. 
 
-![Remix IDE Toolbar](/assets/lessons/1/img_1.png)
+![Remix IDE Toolbar](/public/assets/lessons/1/img_1.png)
```

Comment:
These additions of `public/` seem to have broken the rendering of those images.

Reply:
I think it actually fixed them if you check the `fix` at  https://github.com/losMacos/school-of-code/blob/fundamentals/lessons/1.mdx compared to the original at https://github.com/Developer-DAO/school-of-code/blob/main/lessons/1.mdx